Compile errors

### Tested Godot versions

godot-cpp 4.2.2
godot-cpp 4.2.1

### Tested Godot-DragonBones versions

2.0.1

### System information

Windows 11, i5 11400H, RTX 3050 mobile, 8gb RAM

### Issue description

While compiling with debug or release got errors
```
=====
src\slot.cpp: In lambda function:
src\slot.cpp:24:43: error: 'create_empty' is not a member of 'godot::Image'
   24 |                 Ref<Image> image = Image::create_empty(2, 2, false, Image::FORMAT_RGBA8);
      |                                           ^~~~~~~~~~~~

=====
scons: *** [src\slot.o] Error 1
scons: building terminated because of errors.
```

### Steps to reproduce

-

### Minimal reproduction project (MRP)

-

--- a/src/slot.cpp
+++ b/src/slot.cpp
@@ -3,8 +3,6 @@
 #include <dragonBones/armature/DeformVertices.h>
 #include <dragonBones/model/DisplayData.h>
 #include <dragonBones/model/DragonBonesData.h>
-#include <godot_cpp/classes/image.hpp>
-#include <godot_cpp/classes/image_texture.hpp>
 
 #include "armature.h"
 #include "mesh_display.h"
@@ -20,13 +18,7 @@ Ref<Texture2D> Slot_GD::get_texture() const {
 		}
 	}
 
-	static Ref<Texture2D> empty_texture{ [] {
-		Ref<Image> image = Image::create_empty(2, 2, false, Image::FORMAT_RGBA8);
-		image->fill(Color(0.0f, 0.0f, 0.0f, 0.0f));
-		Ref<ImageTexture> ret = ImageTexture::create_from_image(image);
-		return ret;
-	}() };
-	return empty_texture;
+	return {};
 }
 
 void Slot_GD::_updateZOrder() {

